sdk/communication/azure-communication-administration/src/main/java/com/azure/communication/administration/PhoneNumberAsyncClient.java

```diff
@@ -80,10 +94,8 @@
     @ServiceMethod(returns = ReturnType.SINGLE)
     public Mono<AreaCodes> getAllAreaCodes(
```

Comment:
Not related to the immediate change, but wondering why these methods are not following the 'list' paradigm, like listAreaCodes, listAreaCodesWithResponse etc.?

Reply:
The main difference here the return value. AreaCodes is a composite object that contains data. It is not a pageable list in this case, so we are not using the list* convention. 

To avoid confusion, perhaps we can rename it to "getAreaCodes()" instead in a separate PR.